src/esp/assets/ResourceManager.cpp

```diff
@@ -1984,6 +1988,119 @@ void ResourceManager::removePrimitiveMesh(int primitiveID) {
   primitive_meshes_.erase(primitiveID);
 }
 
+bool ResourceManager::loadAsset(
+    const std::string& assetName,
+    bool requiresLighting,
+    const esp::assets::PhongMaterialColor* materialOverride,
+    scene::SceneNode* parent,
+    DrawableGroup* drawables,
+    std::vector<scene::SceneNode*>* visNodeCache) {
+  std::string finalAssetName = assetName;
+  bool meshSuccess = true;
+  if (resourceDict_.count(assetName) == 0) {
+    esp::assets::AssetInfo meshinfo{AssetType::UNKNOWN, assetName};
+    meshinfo.requiresLighting = requiresLighting;
+    meshSuccess = loadRenderAssetGeneral(meshinfo);
```

Comment:
Is there a good reason to use the more narrow loadRenderAssetGeneral here versus the more broad loadRenderAsset?